examples/internal/clients/abe/api/swagger.yaml

```diff
@@ -2053,7 +2052,7 @@ paths:
         name: "body"
         required: true
         schema:
-          $ref: "#/definitions/examplepbABitOfEverything"
+          $ref: "#/definitions/A bit of everything_1"
```

Comment:
Title/summary instead of name!

Reply:
How is this generated? `a_bit_of_everything.swagger.json` lists the properties. Is it an assumption in json -> yaml that body parameters always has a schema ref?

Reply:
I think this yaml file is generated from the swagger.json source of truth.

Reply:
This is generated by `swagger-codegen`. I'll investigate if this is a bug with that program or if the generated swagger.json contains wrong.

Reply:
Aha - `swagger-codegen` moves the inline schema to the `definitions:` section, and uses the title to generate the reference. All good then!